docs/modules/ROOT/pages/client.adoc

```diff
@@ -170,6 +170,16 @@ include::./includes/additional-request-args.adoc[leveloffset=+1, opts=optional]
 
 include::./includes/bean-validation.adoc[leveloffset=+1, opts=optional]
 
+[[generate-apis]]
+== Generate APIs
+
+include::./includes/generate-apis.adoc[leveloffset=+1, opts=optional]
```

Comment:
Why do we need an include here?

Reply:
I was following way the bean-validation was included. I can switch it up though.